modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/RoleManager.java

```diff
@@ -135,4 +135,80 @@ default Role patchRole(String roleId, Map<String, List<PatchOperation>> patchOpe
 
         throw new NotImplementedException("patchRole method is not implemented");
     }
+
+    /**
+     * Assign/De-assign users from the role.
+     *
+     * @param oldRole Old role.
+     * @param newRole New role.
+     * @return Updated role.
+     * @throws NotImplementedException
+     * @throws BadRequestException
+     * @throws CharonException
+     * @throws ConflictException
+     * @throws NotFoundException
```

Comment:
complete the comments.
Fix other places as well

Reply:
Fixed with: https://github.com/wso2/charon/pull/435/commits/a1141ae25e587f7b8ac5d5275085cde5ea17c879